man/systemd.network.xml

```diff
@@ -3208,7 +3216,30 @@
         <term><varname>Priority=</varname></term>
         <listitem>
           <para>Specifies the priority of the class. In the round-robin process, classes with the lowest
-          priority field are tried for packets first. This setting is mandatory.</para>
+          priority field are tried for packets first.</para>
+        </listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>Quantum=</varname></term>
+        <listitem>
+          <para>Specifies how much bytes to serve from leaf at once.</para>
```

Comment:
s/much/many/

Reply:
the unit is bytes? hence should be called QuantumBytes=, no?